src/app/profile/settings/feature-opt-in/feature-opt-in.component.html

```diff
@@ -1,86 +1,126 @@
 <div class="container-fluid container-cards-pf">
-  <div class="col-xs-10 col-sm-6">
-    <div class="card-pf list-group form-group list-view-pf list-view-pf-view">
-      <label class="feature-opt-in-label" for="released">
-        <div class="row">
-          <div class="list-group-item">
-            <div class="list-view-pf-checkbox">
-              <input id="released" type="radio" name="No Pre-Production Features" value="released" [(ngModel)]="featureLevel" (ngModelChange)="updateProfile()">
+  <div class="col-xs-10 col-sm-12">
+      <pfng-list
+        [config]="listConfig"
+        [expandTemplate]="untoggledTemplate"
+        [items]="items"
+        [itemTemplate]="itemTemplate">
+
+
+        <ng-template #itemTemplate let-item="item">
+          <div *ngIf="item.displayed" class="list-pf-left list-pf-content-flex">
+            <pfng-list-expand-toggle [expandId]="item.name" [item]="item" [template]="betaFeatureTemplate">
+              <ng-template #betaFeatureTemplate>
+              </ng-template>
+            </pfng-list-expand-toggle>
+          </div>
+          <div *ngIf="item.displayed" class="checkbox list-pf-left list-pf-content-flex">
+            <input id="{{item.name}}" type="radio" name="{{item.title}}" value="{{item.name}}" [(ngModel)]="featureLevel" (ngModelChange)="updateProfile()">
+          </div>
+          <div *ngIf="item.displayed" class="list-pf-main-content">
+            <div class="list-pf-left">
+              <span *ngIf="item.name === 'beta'" class="fa list-pf-icon list-pf-icon-bordered list-pf-icon-small" > ß </span>
+              <span *ngIf="item.name !== 'beta'" class="fa {{item.icon}} list-pf-icon list-pf-icon-bordered list-pf-icon-small"></span>
+            </div>
+            <div class="list-pf-main-content">
+              <div>
+                <p class="feature-title">{{item.title}}</p>
+                <p class="feature-description">{{item.description}}</p>
+              </div>
             </div>
-            <div class="list-view-pf-main-info">
-              <div class="list-view-pf-left">
-                <span class="fa fa-globe list-view-pf-icon-sm"></span>
+          </div>
+        </ng-template>
+
+        <ng-template #untoggledTemplate let-item="item" let-index="index">
+          <div *ngIf="item.expandId === 'released'">
+            <div class="col-sm-1">
```

Comment:
As this and the next `<div>` are `col-sm-*`, but the last is `col-xs-9`, when collapsed to the `xs` bucket, you will have more than 12 columns.